releases/v2.2.0-alpha.20190211.md

```diff
@@ -0,0 +1,215 @@
+---
+title: What&#39;s New in v2.2.0-alpha.20190211
+toc: true
+summary: Additions and changes in CockroachDB version v2.2.0-alpha.20190211 since version v2.2.0-alpha.20190114
+---
+
+## February 11, 2019
+
+In addition to SQL language enhancements, general usability improvements, performance improvements, and bug fixes, this release includes several major highlights:
+
+- **Follower Reads:** Enterprise users can now reduce read latencies by allowing queries to perform historical reads of the closest replica of a given piece of data rather than reading from the more distant "leaseholder" replica. To enable follower reads on a query, use the `experimental_follower_read_timestamp()` built-in function in conjunction with the `AS OF SYSTEM TIME` clause.  
+- **Cost-Based Optimizer (CBO):** The cost-based optimizer now supports almost all read-only queries (except window functions) and almost all mutations (e.g., `CREATE TABLE AS`, `INSERT`, `UPDATE`, `UPSERT`, `DELETE`). In addition, the CBO now reorders up to 4 joins in a query to attempt to find the most performant ordering (via the the new `experimental_reorder_joins_limit` [session variable](../v2.2/set-vars.html)) and takes advantage of automatic statistics without impacting foreground traffic. Note that statistics are created by default on all indexed columns when a user upgrades to this version. Finally, a query plan cache now saves a portion of the planning time for frequent queries used in the cost-based optimizer.
```

Comment:
@jseldess re: where to add links to the CBO highlight, I think the first use of "cost-based-optimizer" can be made a link to the [Cost-based Optimizer](https://www.cockroachlabs.com/docs/dev/cost-based-optimizer.html) page, since all doc updates related to the changes mentioned here will be landing there over the next few days.

Reply:
Done.